zboxcore/sdk/allocation.go

```diff
@@ -392,6 +395,10 @@ func (a *Allocation) uploadOrUpdateFile(localpath string, remotepath string,
 		uploadReq.fullconsensus = float32(bits.TrailingZeros32(uploadReq.uploadMask + 1))
 	}
 
+	if !uploadReq.IsFullConsensusSupported() {
+		return fmt.Errorf("allocation requires a greater number of blobbers than is supported. reduce number of data or parity shards and try again")
```

Comment:
Requesting input on error message

Reply:
Looks good, however i'd suggest that the error message should also advise the actual limit.

Reply:
07f3f8837df504d2f5f164992b34f4542f60810e